doc/sphinx/whats-new/changes-7.6.rst

```diff
@@ -25,14 +25,14 @@ varnishd
 
 A new ``linux`` jail has been added (configured via the ``-j`` argument) which is
 now the default on Linux. For now, it is almost identical to the ``unix`` jail
-with one :ref:`whatsnew_upgrading_CURRENT_linux_jail` added.
+with one :ref:`whatsnew_upgrading_7.6_linux_jail` added.
 
 The port of a *listen_endpoint* given with the ``-a`` argument to ``varnishd``
 can now also be a numerical port range like ``80-89``, besides the existing
 options of port number (e.g. ``80``) and service name (e.g. ``http``). With a
 port range, Varnish will accept connections on all ports within the range.
```

Comment:
Is the range inclusive? I think it should tell here even if it is not the reference manual.

Reply:
for (i = lo; i <= hi && !error; i++) {

Its inclusive and thats the only thing that makes sense?

Reply:
Not sure it's the only one, I had to pause and wonder.